demos/background_subtraction_demo/cpp_gapi/README.md

```diff
@@ -0,0 +1,134 @@
+# G-API Background Subtraction Demo
+
+This demo shows how to perform background subtraction using G-API.
```

Comment:
```suggestion
# G-API Background Subtraction Demo

![example](../background_subtraction.gif)

This demo shows how to perform background subtraction using G-API.
```

The same relative path should be also updated for python README. Please, could you move the gif from python folder to the parent one? In this case the gif will be showed for both versions in the documentation.

Reply:
In fact, G-API demo doesn't have `--show_with_original_frame`, so it's impossible to generate exactly the same `*.gif` using this demo. Let it be without `*.gif`

Reply:
Yeah, I did not take that into account. Okay